README.md

```diff
@@ -22,10 +22,10 @@ If you would like to hack on <REPO_NAME>, please check out the [Snake Charmers
 Tactical Manual](https://github.com/ethereum/snake-charmers-tactical-manual)
 for information on how we do:
 
--   Testing
--   Pull Requests
--   Code Style
--   Documentation
+- Testing
+- Pull Requests
+- Code Style
+- Documentation
```

Comment:
heh, this was my editor auto-formatting markdown. are you using a separate automated tool or 'manual linting'?

Reply:
Was reported error: MD030/list-marker-space: Spaces after list markers [Expected: 1; Actual: 3] markdownlint(MD030)